plonky3/src/params/baby_bear.rs

```diff
@@ -20,10 +20,6 @@ use rand::{distributions::Standard, Rng, SeedableRng};
 use powdr_number::BabyBearField;
 
 const D: u64 = 7;
-// params directly taken from plonky3's poseidon2_round_numbers_128 function
-// to guarantee 128-bit security.
-const ROUNDS_F: usize = 8;
-const ROUNDS_P: usize = 13;
```

Comment:
Didn't really see a reason to hard-code this here. If for any reason the `poseidon2_round_numbers_128`  function changes, we'll want to use the updated numbers.

Reply:
@Schaeff LMK if you had a reason I'm not aware of :)

Reply:
I don't think there is a reason for it, I did not write this code but missed that in the review.